index.js

```diff
@@ -1,21 +1,3 @@
-/* global Blob, FileReader */
+/* global Response, Buffer */
 
-module.exports = function blobToBuffer (blob, cb) {
-  if (typeof Blob === 'undefined' || !(blob instanceof Blob)) {
-    throw new Error('first argument must be a Blob')
-  }
-  if (typeof cb !== 'function') {
-    throw new Error('second argument must be a function')
-  }
-
-  const reader = new FileReader()
-
-  function onLoadEnd (e) {
-    reader.removeEventListener('loadend', onLoadEnd, false)
-    if (e.error) cb(e.error)
-    else cb(null, Buffer.from(reader.result))
-  }
-
-  reader.addEventListener('loadend', onLoadEnd, false)
-  reader.readAsArrayBuffer(blob)
-}
+module.exports = anything => new Response(anything).arrayBuffer().then(Buffer.from)
```

Comment:
I would prefer this function to be named so it shows up with a name in stack traces. Also, I know that `Response` can handle many types, but the point of this package is to convert a blob, so the argument should be called `blob`.

Reply:
ok